cargo-dist/src/init/v0.rs

```diff
@@ -258,7 +99,7 @@ pub fn do_init(cfg: &Config, args: &InitArgs) -> DistResult<()> {
             .interact()?;
 
         if is_migrating {
-            do_migrate()?;
+            migrate::do_migrate()?;
```

Comment:
this makes me think we could have `migrate::do` but this type of renaming isn't super important atm